packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue

```diff
@@ -39,8 +52,13 @@ defineSlots<{
   @include checkbox.variables();
 
   @include layers.component() {
+    --onyx-listbox-option-color: var(--onyx-color-text-icons-neutral-intense);
+    --onyx-listbox-option-color-active: var(--onyx-color-text-icons-primary-intense);
```

Comment:
Should we rename this to --onyx-listbox-options-color-selected, so it's aligned with the background-selected?

Reply:
The color is only applied when the option is active, right? Not when its only selected?

Reply:
I think it's applied when it's selected and active